[2022-12-28T08:05:58.805+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-28T08:05:53.830871+00:00 [queued]>
[2022-12-28T08:05:58.849+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-28T08:05:53.830871+00:00 [queued]>
[2022-12-28T08:05:58.849+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-28T08:05:58.850+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-28T08:05:58.851+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-28T08:05:59.227+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_table> on 2022-12-28 08:05:53.830871+00:00
[2022-12-28T08:05:59.251+0000] {standard_task_runner.py:55} INFO - Started process 3326 to run task
[2022-12-28T08:05:59.274+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'new_table_and_data_import_dag_v1', 'create_table', 'manual__2022-12-28T08:05:53.830871+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/new_table_and_data_import_dag.py', '--cfg-path', '/tmp/tmpmnbbhm7e']
[2022-12-28T08:05:59.283+0000] {standard_task_runner.py:83} INFO - Job 16: Subtask create_table
[2022-12-28T08:05:59.925+0000] {task_command.py:389} INFO - Running <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-28T08:05:53.830871+00:00 [running]> on host 016cb28b89a1
[2022-12-28T08:06:00.715+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=new_table_and_data_import_dag_v1
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-28T08:05:53.830871+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-28T08:05:53.830871+00:00
[2022-12-28T08:06:00.719+0000] {sql.py:253} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS test_table (
                ticker varchar(3),
                year integer,
                quarter integer,
                invest_cost integer,
                from_invest integer,
                from_financial integer,
                from_sale integer,
                free_cash_flow double precision,
                PRIMARY KEY (ticker, year, quarter),
                FOREIGN KEY (ticker) REFERENCES listing_companies (ticker)
            );
        
[2022-12-28T08:06:00.755+0000] {base.py:73} INFO - Using connection ID 'financial_database' for task execution.
[2022-12-28T08:06:01.845+0000] {base.py:73} INFO - Using connection ID 'financial_database' for task execution.
[2022-12-28T08:06:01.905+0000] {sql.py:364} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS test_table (
                ticker varchar(3),
                year integer,
                quarter integer,
                invest_cost integer,
                from_invest integer,
                from_financial integer,
                from_sale integer,
                free_cash_flow double precision,
                PRIMARY KEY (ticker, year, quarter),
                FOREIGN KEY (ticker) REFERENCES listing_companies (ticker)
            );
        , parameters: None
[2022-12-28T08:06:01.968+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=new_table_and_data_import_dag_v1, task_id=create_table, execution_date=20221228T080553, start_date=20221228T080558, end_date=20221228T080601
[2022-12-28T08:06:02.180+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-28T08:06:02.309+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
